unit_tests/test_molecule_interfaces.py

```diff
@@ -230,6 +238,38 @@ def test_to_smiles_and_from_smiles_requires_rdkit_package(self, plams_mols, smil
             with pytest.raises(MissingOptionalPackageError):
                 from_smiles(smiles[0])
 
+    def test_to_image_and_get_reaction_image_can_generate_img_files(self, xyz_folder):
```

Comment:
I have converted the attached example to a unit test.

It's not too fancy, just makes sure the image files can be generated, but then they can be visually inspected too.